src/spatial/models/active_fire_model.py

```
@@ -0,0 +1,521 @@
+        except (TypeError, ValueError):
+            raise ActiveFireValidationError("limit must be a positive integer.")
+        if limit < 1:
+            raise ActiveFireValidationError("limit must be a positive integer.")
+        return limit
+
+    @staticmethod
+    def _validate_hours(hours) -> int:
+        try:
+            hours = int(hours)
+        except (TypeError, ValueError):
+            raise ActiveFireValidationError("hours must be an integer from 1 to 120.")
```

Ruff (0.15.17)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)